persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GenesetMapper.xml

```diff
@@ -44,14 +44,14 @@
         WHERE geneset.EXTERNAL_ID = #{genesetId}
     </select>
     
-    <select id="getGenesetsByGenesetIds" resultType="org.cbioportal.model.Geneset">
+    <select id="fetchGenesets" resultType="org.cbioportal.model.Geneset">
         SELECT
         <include refid="select">
             <property name="prefix" value=""/>
         </include>
         FROM geneset
         WHERE geneset.EXTERNAL_ID IN
-        <foreach item="item" collection="genesetIds" open="(" separator="," close=")">
+        <foreach item="item" collection="list" open="(" separator="," close=")">
```

Comment:
@ersin do you know why this one is needed?

Reply:
I would expect the original one to just work...no?

Reply:
@pieterlukasse there is a problem in mybatis library when using Java 8 and omitting `@Param`, if there is only one parameter in the method, you need to name it `_parameter` if a single object or `list` if it's list, instead of their original names.

Reply:
ok, thanks